asn1/GenericTypes.asn

```diff
@@ -0,0 +1,66 @@
+GenericTypes
+DEFINITIONS
+AUTOMATIC TAGS ::=
+BEGIN
+
+-- Small Integers
+Int8 ::= INTEGER (-128..127)
```

Comment:
Do we need all of these types anymore? I appreciate the value of the comments for documenting how the design was finalized but perhaps we can put that somewhere else to keep this clean?